x/concentrated-liquidity/math/math_test.go

```diff
@@ -169,6 +176,15 @@ func (suite *ConcentratedMathTestSuite) TestCalcAmount0Delta() {
 			isWithTolerance: false,
 			// https://www.wolframalpha.com/input?i=%281517882343.751510418088349649+*+%2874.161984870956629487+-+70.710678118654752440+%29%29+%2F+%2870.710678118654752440+*+74.161984870956629487%29
 		},
+		"happy path, sqrtPriceA greater than sqrtPrice B": {
+			liquidity:       sdk.MustNewDecFromStr("1517882343.751510418088349649"), // we use the smaller liquidity between liq0 and liq1
+			sqrtPA:          sdk.MustNewDecFromStr("74.161984870956629487"),         // 5500
+			sqrtPB:          sqrt5000,                                               // 5000
```

Comment:
In a subsequent PR, we should just generically permute this to remove the duplication